docs/content/en/docs/contribution-guidelines/control-plane-development.md

```diff
@@ -24,4 +26,146 @@ bazelisk run //cmd/server:server -- server \
+        address: 0.0.0.0
+        port_value: 9090
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          access_log:
+            name: envoy.file_access_log
+            config:
+              path: /dev/stdout
+            filter:
+              not_health_check_filter: {}
+          codec_type: auto
+          idle_timeout: 600s
+          stat_prefix: ingress_http
+          http_filters:
+          - name: envoy.grpc_web
+          - name: envoy.cors # If cors is enable
+          - name: envoy.router
+          route_config:
+            virtual_hosts:
+            - name: envoy
+              domains:
+                - '*'
+              cors:
+                allow_origin:
+                - http://localhost:9090
+                allow_methods: GET, PUT, DELETE, POST, OPTIONS
+                allow_headers: keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,custom-header-1,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout,authorization
+                allow_credentials: true
+                max_age: "1728000"
+                expose_headers: custom-header-1,grpc-status,grpc-message
+              routes:
+                - match:
+                    prefix: /pipe.api.service.pipedservice.PipedService/
+                    grpc:
+                  route:
+                    cluster: server-piped-api
+                - match:
+                    prefix: /pipe.api.service.webservice.WebService/
+                    grpc:
+                  route:
+                    cluster: server-web-api
+                - match:
+                    prefix: /
+                  route:
+                    cluster: server-http
+  clusters:
+    - name: server-piped-api
+      http2_protocol_options: {}
+      connect_timeout: 0.25s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: server-piped-api
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: localhost
+                  port_value: 9080
+    - name: server-web-api
+      http2_protocol_options: {}
+      connect_timeout: 0.25s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: server-web-api
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: localhost
+                  port_value: 9081
+    - name: server-http
+      connect_timeout: 0.25s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: server-http
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: localhost
+                  port_value: 9082
+```
```

Comment:
nit: Maybe we should add a `local` directory at the root of the repository to store those files (`envoy-config.yaml`, `Dockerfile`, `piped-config.yaml`).